lib/invoices/utils.ts

```diff
@@ -86,10 +85,9 @@ export const generateApplicationInvoicePdf = (
     },
   });
 
-  const pdfDoc = printer.createPdfKitDocument(definition);
-  // TODO: Upload invoice to S3
-  pdfDoc.pipe(fs.createWriteStream('temp/invoice.pdf'));
-  pdfDoc.end();
+  const pdfDocument = printer.createPdfKitDocument(definition);
+  pdfDocument.end();
+  return pdfDocument;
```

Comment:
s3 sdk actually accepts a readable stream so we can skip saving the file to the server